test/container_test.py

```diff
@@ -1423,6 +1424,23 @@ async def custom_heartbeater(self):
     assert "Traceback" not in caplog.text  # should not print a full traceback - don't scare users!
 
 
+@skip_github_non_linux
+@pytest.mark.usefixtures("server_url_env")
+def test_container_doesnt_send_large_exceptions(servicer):
+    ret = _run_container(
```

Comment:
Maybe add a comment to this test (and changed code in the container_io_manager) to so we know what it tests/prevents.